rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The rules were too broad in WR-135 because the class names were used in the title of the rule ... generating a large amount of noise.

With those rules, the effort level is "0" (pure information) and all entries are bundled together.